Modules/_datetimemodule.c

```diff
@@ -6333,9 +6333,18 @@ local_to_seconds(int year, int month, int day,
             u2 = u1 + max_fold_seconds;
         else
             u2 = u1 - max_fold_seconds;
+#ifdef MS_WINDOWS
+        u2 += max_fold_seconds;
```

Comment:
I _think_ we only want to do this if it's too small? Otherwise it has the potential to do the conversion wrong if you put in a "normal" time that's within `max_fold_seconds` of an offset change (either DST or legal change). We probably assume that none of these happened on the first day after epoch.

Otherwise, this seems like a fine way to handle it. A "real" fix would be much more complex, and probably inconsistent with actual timezone aware datetimes, which I'm sure are the preferred approach these days.

Reply:
I agree with @zooba. See the Windows-specific comment in `datetime_from_timet_and_us()`.

Also this change needs tests.